core/controllers/concept_card_viewer.py

```diff
@@ -31,7 +31,7 @@ class ConceptCardDataHandler(base.BaseHandler):
     @acl_decorators.can_view_skills
     def get(self, comma_separated_skill_ids):
         """Handles GET requests.
-
+        
```

Comment:
Please remove unwanted space. (Check how to enable trailing whitespace remove on save feature for your IDE)

Reply:
it's the pattern ( requirement )  only not an extra space!

Reply:
Isn't the requirement just a newline here and not empty spaces? Probably this throws a lint error as well. can you remove it?